elyra/pipeline/pipeline.py

```diff
@@ -487,6 +487,24 @@ def merge(cls, primary: "KeyValueList", secondary: "KeyValueList") -> "KeyValueL
 
         return KeyValueList.from_dict({**secondary_dict, **primary_dict})
 
+    @classmethod
+    def difference(cls, minuend: "KeyValueList", subtrahend: "KeyValueList") -> "KeyValueList":
+        """
+        Given two lists, convert to dictionaries and remove any keys found in the
+        subtrahend from the minuend, if present.
```

Comment:
```suggestion
        Given two KeyValueLists, convert to dictionaries and remove any keys found in the
        second (subtrahend) from the first (minuend), if present.
```
By adding a positional notation, it might help folks with the unusual (albeit correct) terminology.